packages/shared/dao/types.ts

```diff
@@ -33,19 +32,14 @@ type BaseCandidate = {
   catalystVersion: string
 }
 
-export type LayerBasedCandidate = {
-  type: 'layer-based'
-  layer: Layer
-} & BaseCandidate
-
 export type IslandsBasedCandidate = {
   type: 'islands-based'
   usersCount: number
   usersParcels?: Parcel[]
   maxUsers?: number
 } & BaseCandidate
 
-export type Candidate = LayerBasedCandidate | IslandsBasedCandidate
+export type Candidate = IslandsBasedCandidate
```

Comment:
Maybe we can remove the type `IslandBasedCandidate`?

Reply:
👍